com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -261,54 +261,58 @@ private void OnValidate()
                 };
             }
 
-            // During OnValidate we will always clear out NetworkPrefabOverrideLinks and rebuild it
-            NetworkConfig.NetworkPrefabOverrideLinks.Clear();
-
-            // Check network prefabs and assign to dictionary for quick look up
-            for (int i = 0; i < NetworkConfig.NetworkPrefabs.Count; i++)
+            // If the scene is dirty and the asset database is not currently updating then we can update NetworkPrefab information
+            if (activeScene.isDirty && !UnityEditor.EditorApplication.isUpdating)
```

Comment:
IDK why Git is confused about the diff but this part is the only diff we actually have — and it looks good to me! 🚀

Reply:
I believe it is because all of the indentions and line numbers changed when I enclosed all of the Network Prefab list stuff within that if statement.

Reply:
It looks good to me, the only thing I am slightly worried about, is whether we can use a conditional early return/guard instead of enclosing all the code below, it makes it less readable given the Identations.